validator/accounts/v2/wallet_create_test.go

```diff
@@ -233,7 +233,8 @@ func TestCreateWallet_WalletAlreadyExists(t *testing.T) {
 
 	// We attempt to create another wallet of the same type at the same location. We expect an error.
 	_, err = CreateAndSaveWalletCli(cliCtx)
-	require.NotEqual(t, nil, err,
+	require.ErrorContains(t, "a wallet of this type already exists at this location. Please input an"+
+		" alternative location for the new wallet or remove the current wallet", err,
 		"Creating a wallet of the same type at an existing wallet location should return error")
```

Comment:
This argument can be omitted as it does not add much value.

Reply:
kQBWkNKqzCAwA9vjMwRmg7aY75Rf8lByPA9zKXoqGkHi8SM7